### DISCUSSION TITLE
买了两年的专业版，一直没法收到邮件验证码

### DISCUSSION THREAD

**Discussion by xelmirage** [getlantern/lantern-forum-cn#1003] [General]
邮箱是outlook,麻烦换个渠道给我发邮件，或者给我退款，我换个邮箱再买，谢谢

**Comment by MaxMKMax** [comment]
@xelmirage 您好. 谢谢您通知我. 我已经帮您提交问题上去了. 这几天会尽快给您处理.

**Comment by MaxMKMax** [comment]
@xelmirage 您好. 请问你有更细节的单据吗?
